cltk/corpus/latin/__init__.py

```diff
@@ -17,7 +17,7 @@
 # Would like to have this search through a CLTK_DATA environment variable
 # Better to use something like make_cltk_path in cltk.utils.file_operations?
 home = os.path.expanduser('~')
-cltk_path = os.path.join(home, 'cltk_data')
+cltk_path = os.path.join(home, 'CLTK_DATA')
```

Comment:
This is a mistake, which you overwrote from an old fork. Follow the Git workflow examples again and update your fork using a remote.